models/volunteer.py

```diff
@@ -0,0 +1,95 @@
+import datetime
+from models import camp
+
+
+class Volunteer:
+    """
+    A class used to represent a volunteer.
+    """
+
+    def __init__(self, firstname: str,
+                 lastname: str,
+                 phone: str,
+                 volunteercamp: list,
+                 availability = True):
+        """
+           Initialize a new volunteer.
+           :param firstname: the firstname of the volunteer
+           :param lastname: the lastname of the volunteer
+           :param phone: the phone number of the volunteer
+           :param volunteercamp: the camp that the volunteer belongs to
+           :param availability:  whether the volunteer is available to join a new emergency plan
+           :param creationdate: the date that this volunteer is created
+           """
+        currentdate = datetime.datetime.now()
+
+        self.name = self.__checkVolunteerName(firstname, lastname)
+        self.phone = self.__checkVolunteerPhone(phone)
+        self.camp = volunteercamp
+        self.availability = availability
+        self.creationdate = currentdate.date()
+
+
+    def __checkVolunteerName(self, firstname, lastname):
+        if len(firstname) <= 1:
+            raise self.InvalidFirstnameException(firstname)
+        elif len(lastname) <= 1:
+            raise self.InvalidLastnameException(lastname)
+        name = f"({firstname}+' '+{lastname})"
+        return name
+
+    def __checkVolunteerPhone(self, phone):
+        if phone[0] != "+":
+            raise self.InvalidPhoneException("Phone should include country code and be in format \"+XXXXXXXXXXX\".")
+        elif len(phone) < 5:
+            raise self.InvalidPhoneException("The phone number is too short.")
```

Comment:
You may actually validates it with [regex](https://stackoverflow.com/questions/2113908/what-regular-expression-will-match-valid-international-phone-numbers).
The regex library from core-python is `re`

Reply:
Thanks for the suggestion! will consider using in the new feature "volunteerAccount"